docs/reference/cli.md

```diff
@@ -6026,6 +6026,67 @@ uv pip compile [OPTIONS] <SRC_FILE>...
 <p>By default, uv uses the virtual environment in the current working directory or any parent directory, falling back to searching for a Python executable in <code>PATH</code>. The <code>--system</code> option instructs uv to avoid using a virtual environment Python and restrict its search to the system path.</p>
 
 <p>May also be set with the <code>UV_SYSTEM_PYTHON</code> environment variable.</p>
+</dd><dt id="uv-pip-compile--torch-backend"><a href="#uv-pip-compile--torch-backend"><code>--torch-backend</code></a> <i>torch-backend</i></dt><dd><p>The backend to use when fetching packages in the <code>PyTorch</code> ecosystem (e.g., <code>cu126</code> or <code>auto</code>)</p>
+
+<p>When set, uv will ignore the configured index URLs for packages in the <code>PyTorch</code> ecosystem, and will instead use the defined backend.</p>
+
+<p>For example, when set to <code>cpu</code>, uv will use the CPU-only <code>PyTorch</code> index; when set to <code>cu126</code>, uv will use the <code>PyTorch</code> index for CUDA 12.6.</p>
+
+<p>The <code>auto</code> mode will attempt to detect the appropriate <code>PyTorch</code> index based on the currently installed CUDA drivers.</p>
+
+<p>Possible values:</p>
```

Comment:
I wonder if it would be helpful to put the full giant list behind a `<summary>...</summary>`.